src/renderer/actions/install.js

```diff
@@ -78,19 +81,32 @@ const getJSON = phin.defaults({method: "GET", parse: "json", followRedirects: tr
 const downloadFile = phin.defaults({method: "GET", followRedirects: true, headers: {"User-Agent": "BetterDiscord Installer", "Accept": "application/octet-stream"}});
 const asarPath = path.join(bdDataFolder, "betterdiscord.asar");
 async function downloadAsar() {
-    let downloadUrl = "https://api.github.com/repos/BetterDiscord/BetterDiscord/releases";
+    const releaseApiUrl = "https://api.github.com/repos/BetterDiscord/BetterDiscord/releases";
```

Comment:
This *might* be better as a top level constant for consistency with other constants in the file. Not a huge deal.

Reply:
I think I'll leave it here for now, it'll have to be moved when we do the refactor we discussed anyways.